scripts/react-namespace-imports.test.mjs

```
@@ -0,0 +1,30 @@
+test('CodePanel uses the required React namespace import', () => {
+	const filename = path.join(ROOT, 'apps', 'template-studio', 'src', 'components', 'CodePanel.tsx');
+	const source = ts.createSourceFile(
+		filename,
+		readFileSync(filename, 'utf8'),
+		ts.ScriptTarget.Latest,
+		true,
+		ts.ScriptKind.TSX
+	);
+	const reactImport = source.statements.find(
+		(statement) =>
+			ts.isImportDeclaration(statement) && statement.moduleSpecifier.text === 'react'
```

ESLint

[error] Replace `⏎↹↹↹` with `·`

(prettier/prettier)